pkg/certificates/cli.go

```diff
@@ -0,0 +1,179 @@
+// +build !no_cert_auth
+
+package certificates
+
+import (
+	"crypto/rsa"
+	"crypto/x509"
+	"fmt"
+	"github.com/project-receptor/receptor/pkg/cmdline"
+	"net"
+	"strings"
+	"time"
+)
+
+type initCA struct {
+	CommonName string `description:"Common name to assign to the certificate" required:"Yes"`
+	Bits       int    `description:"Bit length of the encryption keys of the certificate" required:"Yes"`
+	NotBefore  string `description:"Effective (NotBefore) date/time, in RFC3339 format"`
+	NotAfter   string `description:"Expiration (NotAfter) date/time, in RFC3339 format"`
```

Comment:
not too critical, but we could provide an example of RFC3339 format,
`Effective (NotBefore) date/time, in RFC3339 format, e.g. 2022-07-04T00:00:00.00Z`

Reply:
My only concern about this is that I've been generally trying to keep things so that when you do `receptor --help` it looks good on a standard size (80x25) terminal.  (I haven't always succeeded.)